pkg/apis/submariner.io/v1/types.go

```diff
@@ -100,6 +110,8 @@ type Connection struct {
 	Status        ConnectionStatus `json:"status"`
 	StatusMessage string           `json:"statusMessage"`
 	Endpoint      EndpointSpec     `json:"endpoint"`
+	// +optional
+	LatencySpec LatencySpec `json:"latencySpec"`
```

Comment:
I think the name `Latency` is fine but the type should be `* LatencySpec` since it's optional.

```suggestion
	Latency *LatencySpec `json:"latency,omitempty"`
```